heat/core/factories.py

```diff
@@ -1043,6 +1045,89 @@ def logspace(
     return pow(base, y).astype(dtype, copy=False)
 
 
+def meshgrid(*arrays: Sequence[DNDarray], indexing: str = "xy") -> List[DNDarray]:
+    """
+    Returns coordinate matrices from coordinate vectors.
+
+    Parameters
+    ----------
+    arrays : Sequence[ DNDarray ]
+        one-dimensional arrays representing grid coordinates. If exactly one vector is distributed, the returned matrices will
```

Comment:
"If exactly one vector is distributed..." should be "If one or more vectors are distributed..."

Reply:
one distributed vector is allowed, no more.

Reply:
Otherwise, a `split` parameter will be necessary and the vectors need to be redistributed.